cmd/av/stack_sync.go

```diff
@@ -327,24 +319,58 @@ base branch.
 			})
 
 			parentMeta, ok := branches[parentState.Name]
-			if ok && parentMeta.PullRequest != nil && parentMeta.PullRequest.State == githubv4.PullRequestStateMerged {
+			if ok && parentMeta.MergeCommit != "" {
 				defaultBranch, err := repo.DefaultBranch()
 				if err != nil {
 					return errors.Wrap(err, "failed to determine default branch")
 				}
 				_, _ = fmt.Fprint(os.Stderr,
-					"  - pull request ", colors.UserInput("#", parentMeta.PullRequest.Number),
-					" was merged, this branch should be synced on top of the merge commit:\n",
-				)
-				// TODO:
-				//     We should do this automatically for the user
-				_, _ = colors.CliCmdC.Fprint(os.Stderr,
-					"        git fetch origin ", defaultBranch, ":", defaultBranch, "\n",
-					"        git checkout ", currentBranch, "\n",
-					"        av stack sync --parent ", defaultBranch, "\n",
+					"  - parent pull request ",
+					colors.UserInput("#", parentMeta.PullRequest.Number),
+					" was merged into trunk, syncing branch on merge commit ",
+					git.ShortSha(parentMeta.MergeCommit),
+					"\n",
 				)
-				continue
+				// update the default branch so the merged PR is there
+				_, err = repo.Git("fetch", "origin", fmt.Sprint(defaultBranch, ":", defaultBranch))
+				if err != nil {
+					return err
+				}
+
+				// rebase onto the merge commit from the old parent
+				_, err = repo.Rebase(git.RebaseOpts{
+					Onto:     parentMeta.MergeCommit,
+					Upstream: parentMeta.Name,
+					Branch:   currentBranch,
+				})
+				if err != nil {
+					return err
+				}
+				// now that we have rebased onto trunk - update current branch
+				err = actions.ReparentWriteMetaData(repo, actions.ReparentOpts{
```

Comment:
Oh I missed this. I don't think this is the right function to be calling here. `reparentWriteMetadata` was a private function for a reason. I don't think we should think of this as a reparent operation at all internally. Can we just use the normal `meta.WriteBranch` here?